gf-guide/upstream.md

```diff
@@ -98,7 +98,11 @@ Each file or dir has the following purpose:
 
 -   **[documentation](https://github.com/googlefonts/Unified-Font-Repository/tree/main/documentation)**
     <br>
-    A directory that contains expanded information about the Family. You can eventually store in it your PDF specimen, screenshots or process definition, the pictures you use for the README.md, and [**promotional assets**](marketing.md) for Twitter.
+    A directory that contains expanded information about the Family. You can eventually store in it your PDF specimen, screenshots or process definition, the pictures you use for the README.md. This dir my also include:
```

Comment:
The sentence looks a bit weird, maybe some commas are missing?

Reply:
Please feel free to edit it